### DISCUSSION TITLE
What happens if I Hit "Proceed" after ublock tells me there is an ad tracker

### DISCUSSION THREAD

**Discussion by dmvmoe** [uBlockOrigin/uBlock-issues#2434] [General] [answered]
It appears that any time I go to say homedepot, or even amazon I get ublock telling me before it takes me to amazon that the page has been blocked because of something in "Peter Lowe’s Ad and tracking server list](chrome-extension://cjpalhdlnbpafiamejdnhcphjbkeiagm/asset-viewer.html?url=plowe-0)". If I hit proceed does that mean that basically i'm saying it's okay to let it track?  If that's the case then honestly for what I do on the net I see no need to continue using ublock because I have to go to the website anyway so what's the point of being stopped every single time.   I welcome your thoughts.

**Comment by gorhill** [comment, marked as answer]
> what's the point of being stopped every single time

You can completely disable strict-blocking, it's up to you. See [documentation](https://github.com/gorhill/uBlock/wiki/Strict-blocking).

---

> being stopped every single time

By the way, you are offered the choice to not be warned anymore for the given site, why not just check the box before proceeding?

**Comment by u-RraaLL** [comment]
I'm just curious, what filter does it say stops you from visiting Amazon? Because the site isn't blocked in that lists unless you mean the adsystem domain, then yes, it's blocked and you can unlock it permanently by ticking the checkbox, like gorhill mentioned.

And if what's actually stopping your visits is some type of discount/cashback redirection then yes - you are agreeing to being tracked. To keep exposure to a minimum I usually suggest people using a separate browser profile for such cases.

**Reply by dmvmoe** [reply]
Being tracked.  That's really the part I wanted to know.  I get it and why when I do a search.  If I go directly to the links then no tracking but Amazon is tracking so there is that.  Thank you both for the answers.  I got it now.
